modules/developer_manual/pages/core/apis/provisioning-api.adoc

```diff
@@ -207,6 +207,19 @@ curl http://admin:secret@example.com/ocs/v1.php/cloud/users/Frank
 </ocs>
 ----
 
+==== File Storage Space Quota Response Fields
+
+* free - the number of bytes of quota remaining
+* used - the number of bytes of quota currently used
+* total - the total number of bytes of storage that the user has permission to use
+* relative - the percentage of quota currently used
```

Comment:
```suggestion
free::
The number of bytes of quota remaining

used::
The number of bytes of quota currently used

total::
The total number of bytes of storage that the user has permission to use

relative::
The percentage of quota currently used
```
This renders a bit better. It makes a small headline and underneath the text. Nice if you have a list of items with descriptions.